address.tpl

```diff
@@ -166,8 +181,8 @@
         <input class="validate form-control" data-validate="{$address_validation.phone_mobile.validate}" type="tel" id="phone_mobile" name="phone_mobile" value="{if isset($smarty.post.phone_mobile)}{$smarty.post.phone_mobile}{else}{if isset($address->phone_mobile)}{$address->phone_mobile|escape:'html':'UTF-8'}{/if}{/if}" />
       </div>
     {/if}
-    {if isset($one_phone_at_least) && $one_phone_at_least && !$atLeastOneExists}
-      <p class="help-block required">{l s='You must register at least one phone number.'}</p>
+    {if isset($one_phone_at_least) && $one_phone_at_least && !$atLeastOneExists && !$phoneIsRequired}
```

Comment:
Shouldn't this be `$phoneIsRequired`?

Reply:
seems to work this way..

Reply:
I don't believe that :P 
Let me double check first.